examples/example-gateway/clients/baz/baz.go

```diff
@@ -0,0 +1,237 @@
+// Code generated by zanzibar
+// @generated
+// TODO: (lu) generate
+
+package bazClient
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/uber-go/zap"
+	"github.com/uber/tchannel-go"
+	"github.com/uber/zanzibar/runtime"
+
+	zt "github.com/uber/zanzibar/runtime/tchannel"
+	"go.uber.org/thriftrw/wire"
+
+	"github.com/uber/zanzibar/examples/example-gateway/build/gen-code/github.com/uber/zanzibar/clients/baz/baz"
+)
+
+// TChanBaz is the interface that defines the server handler and client interface.
+type TChanBaz interface {
+	Call(ctx context.Context, reqHeaders map[string]string, r *baz.SimpleService_Call_Args) (map[string]string, *baz.BazResponse, error)
+	Simple(ctx context.Context, reqHeaders map[string]string) (map[string]string, error)
+	SimpleFuture(ctx context.Context, reqHeaders map[string]string) (map[string]string, error)
+}
+
+// BazClient is the client to to talk to its backend service.
+type BazClient struct {
+	SimpleServiceClient
+}
+
+// NewClient returns a new http client for service Bar.
+func NewClient(config *zanzibar.StaticConfig, gateway *zanzibar.Gateway) *BazClient {
+	// TODO: (lu) get clientName from config
+	// clientName := config.MustGetString("clients.baz.clientName")
+	clientName := "BazClient"
+
+	// TODO: (lu) client channel opts
+	ch, err := tchannel.NewChannel(clientName, nil)
+	if err != nil {
+		gateway.Logger.Error("Failed to create tchannel",
+			zap.String("error", err.Error()))
+	}
+
+	ip := config.MustGetString("clients.SimpleService.ip")
+	port := config.MustGetInt("clients.SimpleService.port")
+	ch.Peers().Add(ip + ":" + strconv.Itoa(int(port)))
+
+	// TODO: (lu) get serviceName from config
+	// serviceName := config.MustGetString("clients.baz.serviceName")
```

Comment:
yeah, I removed the comment in the following PR https://github.com/uber/zanzibar/pull/30/files#diff-50f740ccde9b3f6a874ee45ca46b1613L53